Code/client/Services/Generic/ObjectService.cpp

```diff
@@ -44,19 +46,25 @@ ObjectService::ObjectService(World& aWorld, entt::dispatcher& aDispatcher, Trans
 #endif
 }
 
-bool IsVanillaHome(const uint32_t acCellId) noexcept
+bool IsPlayerHome(const TESObjectCELL* pCell) noexcept
 {
-    switch (acCellId)
+
+    if (pCell->loadedCellData->encounterZone)
```

Comment:
Check if loadedCellData is null first

Reply:
As well as pCell, can never be too careful